pages/linux/pacman.md

```diff
@@ -25,3 +25,11 @@
 - list only the explicitly installed packages and versions
 
 `pacman -Qe`
+
+- find which package owns a certain file
+
+`pacman -Qo filename`
+
+- empty package cache to free up space
+
+`pacman -Scc`
```

Comment:
Yeah, of course :) I'm simplifying and I guess curating with this choice a bit. Personally I always use `-Scc` that's why I included that. Looking at [the wiki page](https://wiki.archlinux.org/index.php/Pacman#Cleaning_the_package_cache), it says about `-Scc` that "_It should be avoided unless there is an immediate need for disk space._" I'm only running this command when I see that "oh, I'm out of space, let's see what we can get rid of", so in that sense it does exactly what the label says. So anyways...